bench-tests/bt-0000-0001/README.md

```diff
@@ -8,16 +8,31 @@ random read/write.
 
 The tests that are run using smallfile are, create, read, ls -l (listing)
 
-## Pre-requsites
+## Pre-requisites
   - The environment variable CLIENTS must be set and contain a space separated
-    list of IPs or hostnames
+    list of IPs or hostnames [1]
```

Comment:
I would keep the reference numbers here [a] [b] etc. so that we do not confuse with the references [1] [2] provided at the end of the document.